### PR TITLE
More specific bug description.

### PR DESCRIPTION
**Pull request by chriseth**
@sifmelcara right after merging, I noticed that the description can be a bit more specific.

### CODE REVIEW

File: docs/bugs.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
         "name": "ABIEncoderV2StorageArrayWithMultiSlotElement",
-        "summary": "Storage arrays containing structs or other arrays are not read properly when directly encoded in function calls or in abi.encode*.",
-        "description": "When storage arrays whose elements occupy more than a single storage slot are directly encoded in function calls or using abi.encode*, their elements are read in an overlapping manner, i.e. the element pointer is not properly advanced between reads. This is not a problem when the storage data is first copied to a memory variable.",
+        "summary": "Storage arrays containing structs or other statically-sized arrays are not read properly when directly encoded in function calls or in abi.encode*.",
+        "description": "When storage arrays whose elements occupy more than a single storage slot are directly encoded in function calls or using abi.encode*, their elements are read in an overlapping manner, i.e. the element pointer is not properly advanced between reads. This is not a problem when the storage data is first copied to a memory variable or if the storage array only contains value types, dynamically-sized arrays or mappings.",
```
Comment (sifmelcara) [inline docs/bugs.json]: IIRC we cannot encode mapping type?

Comment (chriseth) [reply docs/bugs.json]: Ah, fair point! :)

**Comment by sifmelcara** [comment]
Also I wonder if there will be a separate article/announcement like https://blog.ethereum.org/2019/03/26/solidity-optimizer-and-abiencoderv2-bug/

If not, IMHO we might want to add more details to the description so that people can be more confident about whether they are affected.

**Comment by chriseth** [comment]
Updated. Also, there will be a blog post.

**Comment by sifmelcara** [comment]
Perhaps replace `function calls` with `external function calls`? (Sorry I didn't notice during last review)

**Comment by chriseth** [comment]
Updated.

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**